assets/src/domain/wpPluginsPage/index.tsx

```diff
@@ -14,9 +14,14 @@ const info = pathOr<ExitSurveyInfo>(null, ['eejsdata', 'data', 'exitModalInfo'],
 const handleDeactivationClick = (e: JQuery.ClickEvent<HTMLElement>, el: any): void => {
 	e.preventDefault();
 	const url = $(el).attr('href');
-	const container = document.getElementById('ee-exit-survey-modal');
+	const container = document.getElementById('wpfooter');
 
-	render(<ExitSurvey deactivationUrl={url} />, container);
+	const root = document.createElement('div');
+	root.id = 'ee-exit-survey-modal';
+
+	container.prepend(root);
+
+	render(<ExitSurvey deactivationUrl={url} />, root);
 };
```

Comment:
if you rebase onto latest EDTR/master you can use `/application/services/utilities/dom/renderDomElement.ts` for this

Reply:
Oh, cool 👍